packages/app-admin/src/components/OptionsMenu/useOptionsMenuItem.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { OptionsMenuItem } from "./OptionsMenuItem";
+
+export interface OptionsMenuItemProviderContext {
+    OptionsMenuItem: typeof OptionsMenuItem;
+}
+
+const OptionsMenuItemContext = React.createContext<OptionsMenuItemProviderContext | undefined>(
+    undefined
+);
+
+interface OptionsMenuItemProviderProps {
+    children: React.ReactNode;
+}
+
+export const OptionsMenuItemProvider = ({ children }: OptionsMenuItemProviderProps) => {
```

Comment:
Add the `React.VFC<OptionsMenuItemProviderProps>`. 
Sometimes you are writing code with the type of the component defined, sometimes you are not. You need to keep it consistent.
By having the React.FC (or VFC), we are adding some base bug safety to our code.